### DISCUSSION TITLE
Node 16 reached its EOL, any ETA for Superset using a supported Node version?

### DISCUSSION THREAD

**Discussion by migruiz4** [apache/superset#28426] [Q&A / Help]
Hi there!

As you may know, node@16 reached its EOL back in September 2023 and we are worried about the security implications it may have to continue using a deprecated version of Node.

I have seen you have merged node 18 into master but there is not a release yet, do you have an ETA for a superset release using it?

Thank you in advance.

**Comment by carrodher** [comment]
Seems the NodeJS bump will be done in 4.1.0 ([changelog rc3](https://github.com/apache/superset/blob/4.1.0rc3/CHANGELOG/4.1.0.md)):
- https://github.com/apache/superset/pull/27198
- https://github.com/apache/superset/pull/27955
- https://github.com/apache/superset/pull/27974
